### PR TITLE
fix(color-picker): [color-picker,color-select-panel] fix issue 1684

### PR DESCRIPTION
**Pull request by GaoNeng-wWw**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #1684 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **New Features**
  - Enhanced color picker functionality with real-time updates to the selected color and alpha values.

- **Bug Fixes**
  - Fixed background style binding issues in the color-picker component.

- **Chores**
  - Reordered dependencies in the color-picker package for better clarity and organization.

- **Refactor**
  - Updated event emission patterns to ensure model values are in sync with user interactions across color selection components.

### CODE REVIEW

File: packages/renderless/src/color-select-panel/index.ts [hunk 1]
```diff
@@ -48,10 +49,13 @@ export const onCancel = (
       })
       emit('cancel', color)
     }
+    // console.log(pre.value)
```
Comment (zzcr) [inline packages/renderless/src/color-select-panel/index.ts]: 调试代码需要删除哈

Comment (kagol) [reply packages/renderless/src/color-select-panel/index.ts]: @GaoNeng-wWw 麻烦解决下检视意见哈

Comment (GaoNeng-wWw) [reply packages/renderless/src/color-select-panel/index.ts]: 好的

Comment (GaoNeng-wWw) [reply packages/renderless/src/color-select-panel/index.ts]: 已修复

**Review by zzcr — APPROVED** [review]

**Review by zzcr — APPROVED** [review]

**Pull request merged**